components/tools/OmeroJava/test/integration/gateway/TablesFacilityTest.java

```diff
@@ -115,7 +119,7 @@ public void testGetTableInfo() throws Exception {
         Assert.assertEquals(info.getColumns(), original.getColumns());
     }
 
-    @Test(dependsOnMethods = { "testAddTable" })
+    @Test(dependsOnMethods = { "testGetTableInfo" })
```

Comment:
This needs to be reverted. Running individual tests, shows correct dependencies.